src/stage1/ir_print.cpp

```diff
@@ -558,7 +558,7 @@ const char* ir_inst_gen_type_str(Stage1AirInstId id) {
         case Stage1AirInstIdWasmMemoryGrow:
             return "GenWasmMemoryGrow";
         case Stage1AirInstIdExtern:
-            return "GenExtrern";
+            return "GenExtern";
```

Comment:
internal?

Reply:
These are just debug output, fixing spelling here is fine.